lib/internal/child_process.js

```diff
@@ -526,6 +527,7 @@ class Control extends EventEmitter {
   constructor(channel) {
     super();
     this.#channel = channel;
+    this.pendingMessages = new Set();
```

Comment:
I don't think we want this to be public.
```suggestion
    this[kPendingMessage] = new SafeSet();
```

Reply:
fixed